cypress/support/helpers/serviceHasPageType.js

```diff
@@ -0,0 +1,5 @@
+import config from '../config/services';
+
+export default (service, pageType) => {
+  return config[service].pageTypes[pageType].paths;
```

Comment:
This will return true for an empty array - makes the function name inaccurate, but probably not a mechanical problem

Reply:
`paths` is an object, so it will either be `null` or have values for each enviroment (either a list of values or an empty array), so a null check should be sufficient here.

Reply:
Or should I combine this with a check for the paths for the Cypress application environment?

Reply:
should this function be returning a boolean?

Reply:
```js
return Boolean(config[service].pageTypes[pageType].paths);
```

Reply:
It's used to filter an array, so should return a boolean. Have reworked this in the most recent commit - to check if the paths.length is > 0

Reply:
I don't think objects have a 'length' property?

Reply:
Oops it's `paths[Cypress.env('APP_ENV')].length`

Reply:
That will throw an exception (cannot read property length of undefined) if a tester haven't provided a value for an environment, eg, if we don't want to run tests on live yet for a page type

Something like '(paths[Cypress.env('APP_ENV')] || []).length' would seem the most stable 🙂

Reply:
This file has changed since the last review - https://github.com/bbc/simorgh/pull/5695/files#diff-1e60ada38d079efacf5e29b923569f5f - it is now using getPaths (https://github.com/bbc/simorgh/pull/5695/files#diff-1aa4c200cf2eecc83470eb1ec7c5c03f), which does implement the logic above.